internal/usecase/interactor/user.go

```diff
@@ -253,12 +266,42 @@ func (i *User) StartPasswordReset(ctx context.Context, email string) error {
 		return err
 	}
 
-	u.SetPasswordReset(user.NewPasswordReset())
+	pr := user.NewPasswordReset()
+	u.SetPasswordReset(pr)
 
 	if err := i.userRepo.Save(ctx, u); err != nil {
 		return err
 	}
 
+	t1, err := textTmpl.New("passwordReset").Parse(passwordResetTextTMPL)
```

Comment:
Make parsing templates in init() and put result in package variable. Another email template should be changed also.

```go
var emailTemplate *text.Template // Be careful of the variable name as it is a package variable

func init() {
	emailTemplate, err := textTmpl.New("passwordReset").Parse(passwordResetTextTMPL)
	if err != nil {
		log.Panicf("password reset email template parse error: %s", err)
	}
}
```